pkg/cleaner/azure/vnetpeering.go

```diff
@@ -0,0 +1,60 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-05-01/network"
+	"github.com/giantswarm/microerror"
+)
+
+// CleanVirtualNetworkPeering delete virtual network peering
+// leftover by e2e test on control plane.
+func (c Cleaner) CleanVirtualNetworkPeering(ctx context.Context) error {
```

Comment:
Why is it exported? Will it be used outside the package?

Reply:
No good point